docs/web3.contract.rst

```diff
@@ -1178,6 +1178,87 @@ Utils
           '_debatingPeriod': 604800,
           '_newCurator': True})
 
+.. py:method:: web3.contract.utils.get_function_info(fn_name, abi_codec, contract_abi, fn_abi, args, kwargs)
+
+    Get function signature and information from a contract ABI given a name, contract 
+    ABI, and arguments.
+
+    If the function exists and its signature matches the provided arguments, the 
+    function ABI, selector and arguments will be returned.
+
+    Inspect a contract function from an ABI as follows:
+
+    .. code-block:: python
```

Comment:
These code blocks would be really good candidates for doctests.  I think there are some examples of doctests in this file, but let me know if you need more direction than that!